robottelo/vm.py

```diff
@@ -69,7 +69,7 @@ def __init__(
                 'clients.provisioning_server')
         else:
             self.provisioning_server = provisioning_server
-        if self.provisioning_server is None:
+        if self.provisioning_server is None or self.provisioning_server == '':
```

Comment:
And this one was found during debugging - if no provisioning server is set in `robottelo.properties`, like this:
```
provisioning_server=
```
then `conf.properties.get('clients.provisioning_server')` is equal to `''`, not `None`. So assertion wouldn't actually be risen in case of empty provisioning server field.